src/System.CommandLine/Parsing/ParseResult.cs

```diff
@@ -85,6 +85,54 @@ public T ValueForOption<T>(string alias)
             }
         }
 
+        [return: MaybeNull]
+        public T ValueForArgument<T>(Argument<T> argument)
+        {
+            if (FindResultFor(argument) is {} result &&
+                result.GetValueOrDefault<T>() is {} t)
```

Comment:
nit: We are not being consistent with the spacing on the pattern `{}` vs `{ }`

Reply:
Formatting consistency is something we might want to set up globally and enforce with `dotnet-format`.